lib/core/utils/get-selector.js

```diff
@@ -18,20 +18,25 @@ const MAXATTRIBUTELENGTH = 31;
  * @return {String}
```

Comment:
Whitespace changes in this file are inconsistent with the rest of the file.

Reply:
don't we have an eslint rule for that?